proof/refine/RISCV64/Invariants_H.thy

```diff
@@ -799,11 +1107,29 @@ where
 lemmas bitmapQ_defs = valid_bitmapQ_def valid_bitmapQ_except_def bitmapQ_def
                        bitmapQ_no_L2_orphans_def bitmapQ_no_L1_orphans_def
 
+(* valid_queues is too strong sometimes *)
+definition valid_inQ_queues :: "kernel_state \<Rightarrow> bool" where
+  "valid_inQ_queues \<equiv> \<lambda>s. \<forall>d p. (\<forall>t\<in>set (ksReadyQueues s (d, p)). obj_at' (inQ d p) t s)"
```

Comment:
I don't think the parentheses around the inner forall are needed.

Reply:
Nope, removed